common-data/src/main/scala/org/genivi/sota/data/Namespace.scala

```diff
@@ -0,0 +1,16 @@
+/**
+ * Copyright: Copyright (C) 2015, Jaguar Land Rover
+ * License: MPL-2.0
+ */
+
+package org.genivi.sota.data
+
+import eu.timepit.refined.api.Refined
+import eu.timepit.refined.auto._
+import eu.timepit.refined.string._
+
+
+object Namespace {
+  // TODO schema for namespace; URI seems reasonable enough
+  type Namespace = String Refined Uri
```

Comment:
URI for namespace ok? that way we cover XML namespaces, e-mails, java-like packages (although not bi-directional) etc.